trustregistry/tests/test_main.py

```diff
@@ -0,0 +1,76 @@
+from unittest.mock import MagicMock, Mock, patch
+
+import pytest
+from fastapi import FastAPI
+from sqlalchemy.orm import Session
+
+from trustregistry import db
+from trustregistry.main import app, lifespan, registry, root
+
+
+@pytest.fixture
+def db_session_mock():
+    session = Mock(spec=Session)
+    return session
```

Comment:
cov report shows these lines are uncovered. I see db_session_mock is indeed used, but not as a fixture argument in the tests